tinygrad/runtime/ops_python.py

```diff
@@ -74,9 +74,11 @@ def __call__(self, *bufs, global_size:Tuple[int,int,int]=(1,1,1), local_size:Tup
         dl[i] = dtype
         if uop is UOps.DEFINE_GLOBAL:
           assert dtype.fmt is not None
+          if sys.version_info[1] < 12: assert dtype.fmt != 'e' and dtype.fmt != '@e'
```

Comment:
Copy pasted with below. Why isn't this needed in the other PR?

Reply:
The way that #7280 was defining DTypeFmtType as a Generic instead of a Literal was probably not allowing mypy to check if memoryview was accepting the half format

Reply:
I removed these asserts because even though python supports half for memoryview in 3.12, the mypy signatures for memoryview still don't support it in version 1.13.0